<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>272</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Keyframe Data</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Target </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="targetLE"/>
        </item>
        <item>
         <widget class="QPushButton" name="selectBTN">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteBTN">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="confirmBTN">
          <property name="text">
           <string>Confirm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Damping</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="keyframeLB">
             <property name="text">
              <string>Keyframe</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="keyframeLE">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Delay</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="delayLE">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Amplitude</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ampLE">
             <property name="text">
              <string>15</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Cycle</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="cycleLE">
             <property name="text">
              <string>0.5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Damping</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dampingLE">
             <property name="text">
              <string>0.046</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Disturbance</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="disturbanceXLE">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="disturbanceYLE">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="disturbanceZLE">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Gravity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="gravityXLE">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="gravityYLE">
             <property name="text">
              <string>-9.8</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="gravityZLE">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Follow</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="followCB">
             <property name="text">
              <string>Follow</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Delay</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="followDelayLE">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Amplitude</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="followAmpLE">
             <property name="text">
              <string>1.0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Parent Object</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="parentObjectLE"/>
           </item>
           <item>
            <widget class="QPushButton" name="pickBTN">
             <property name="text">
              <string>Pick</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Channels</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="rxCB">
           <property name="text">
            <string>Rotate X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ryCB">
           <property name="text">
            <string>Rotate Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="rzCB">
           <property name="text">
            <string>Rotate Z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="txCB">
           <property name="text">
            <string>Translate X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="tyCB">
           <property name="text">
            <string>Translate Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="tzCB">
           <property name="text">
            <string>Translate Z</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
